generic_relations/relations.py

```diff
@@ -44,11 +44,12 @@ def field_to_native(self, obj, field_name):
         """
         value = super(GenericRelatedField, self).field_to_native(
             obj, field_name)
-        serializer = self.determine_deserializer_for_data(value)
+        if value:
```

Comment:
How about `if not value:`?   i have a case where HyperlinkedRelatedSerializer is returning u'' for an empty value in this case.